### ISSUE TITLE
Cannot view comments of public image when not logged in

### ISSUE THREAD

**Issue by Ellen-Wittingen** [csvalpha/amber-api#386]
When viewing a public image, a "401 Unautherized error" is shown in the browser console for the comments. so, for instance, when viewing image 26, which is public, when not logged in, it throws an error when trying to load the url `https://staging.csvalpha.nl/api/v1/photos/26/comments`. However the image is public, so it should allow the not-logged-in user to see the comments.

**Comment by lodewiges** [comment]
I don't think we want comments to be publicly visible because there is no moderation

**Comment by lodewiges** [comment]
okay we also dont want people to download all photo's, which is a possible right now
